src/index.html

```diff
@@ -6,6 +6,7 @@
   <base href="/">
   <link rel="stylesheet" type="text/css" href="assets/index.css">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Please add this locally.. We can't make any remote calls due to privacy

Reply:
Okay